### PR TITLE
ESP32: add setIP and setAPIP

### PR DESCRIPTION
**Pull request by MaBecker**
similar to what I did for ESP8266

### CODE REVIEW

File: libs/network/esp32/jswrap_esp32_network.c [hunk 1]
```diff
@@ -92,6 +92,8 @@ static bool g_isStaConnected = false;
 #define EXPECT_CB_EXCEPTION(jsCB)   jsExceptionHere(JSET_ERROR, "Expecting callback function but got %v", jsCB)
 #define EXPECT_OPT_EXCEPTION(jsOPT) jsExceptionHere(JSET_ERROR, "Expecting Object, got %t", jsOPT)
 
+// Global data structure for setIP and setAPIP
+tcpip_adapter_ip_info_t info;
```
Comment (gfwilliams) [inline libs/network/esp32/jswrap_esp32_network.c:96]: Please could you move this into the setIP function? Looking at tcpip_adapter_set_ip_info I'm pretty sure it doesn't have to be a global variable so having it just in the fn will free up a bit of RAM

**Comment by gfwilliams** [comment]
Looks really good, thanks! Please can you just move the variable mentioned above and I'll get this merged?

**Comment by MaBecker** [comment]
Ups, yes it is no longer needed - thanks for checking!

**Comment by gfwilliams** [comment]
Great, thanks!

**Pull request merged**